src/libraries/System.Linq.Expressions/tests/Dynamic/InvokeMemberBindingTests.cs

```diff
@@ -54,6 +54,7 @@ public static IEnumerable<object[]> ObjectArguments()
             yield return new[] {new object()};
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/55070", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```

Comment:
I think the condition in both cases should be interpreter not iOS/tvOS/MacCatalyst

Reply:
Should I use `PlatformDetection` or, perhaps, wrap ActiveIssue in `#if !FEATURE_COMPILE` ?

Reply:
I think there should be some way to check (even with reflection) that SLE was compiled as IsInterpreting only

Reply:
Added `PlatformDetection.IsLinqExpressionsBuiltWithIsInterpretingOnly` check